### ISSUE TITLE
I've found a few small errors.

### ISSUE THREAD

**Issue by Kofola45cz** [maximuslotro/SignStory#8]
- Testing SignStory v1.0.5 -😄
-------
- When I used SignPicture Edit (Option) on a sign with text, this is what came up:
![Snímek obrazovky 2022-04-24 114614](https://user-images.githubusercontent.com/22054484/164970648-a8fd010a-ed8d-4442-89c2-66840bf93c64.png)
-------
- I don't know if I should take this as a bug, but the File Uploader can be pulled outside of the Minecraft Window.
When minimizing the Minecraft window it does not minimize.
And it can either be in front of or behind the Minecraft window....
I figured out this possible bug when I was trying to see if the bug is fixed (to-do:7).
Understandably the functionality of loud_text is not affected. And after a successful upload, the window closes, so as long as the player doesn't forget to upload the text or close the File Uploader with the Crosshair, so it defacto doesn't matter.
![Snímek obrazovky 2022-04-24 113937](https://user-images.githubusercontent.com/22054484/164971156-5da80d31-6a0c-4ee2-92d0-9429d3123cae.png)

**Comment by maximuslotro** [comment]
1: Not a bug, the original mod would show this menu also when clicking a sign with the "option" option enabled, it allows for different options to show when clicking a valid SignPicture sign, like re-downloading the image or blocking the image, as an added bonus, I found that it allows for copy and pasting signs for non SignPicture signs, by selecting the place option after opening the gui from that menu.. The language path "signpic.gui.settings.load.text" error is a bug and was an issue with the original mod. Easy fix is to add the missing text link to the language file.

2:
1. The file uploader could always be pulled out of the Minecraft window. 
2. Since it acts as a separate window, it wouldn't minimize when Minecraft minimizes.
3. Since it is a separate window, it can be in front of the Minecraft window, and to fix the opener gui from always showing in front of the file selection dialog, I had to make the uploader gui to not always be on top of all windows.
4. It has no effect on load_text functionality, even if it is hidden behind other windows, running the load command again will bring it to the front.

Thanks for the Feedback!  😃 
Since there are no major bugs, i will probably release a fix later in this week.
